tests/internal/py36/test_wrapping.py

```diff
@@ -109,3 +115,59 @@ async def stream():
     assert channel == [b"hello", b""]
     with pytest.raises(StreamConsumed):
         b"".join([_ async for _ in s])
+
+
+@pytest.mark.asyncio
+async def test_wrap_async_generator_throw_close():
```

Comment:
I don't understand why these 3.5 and 3.6 tests are included in this diff. How are they related to 3.11?

Reply:
These are minimum versions. All these tests will run with Python >= 3.5 and >= 3.6 respectively.